%\newpage

\cvsection{\faBuildingO\space Experience}

\begin{cventries}

\cventry
{Postdoctoral Associate - Department of Plant Pathology - Advisor: Jo Handelsman, PhD}
{Wisconsin Institute for Discovery \& University of Wisconsin-Madison}
{Madison, WI}
{06/2019--present}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {\textbf{Advisor:} Jo Handelsman, PhD}
%\item {Genomics and metabolomics of antibiotic discovery and  microbial community interactions.}
%\end{cvitems}
}

\vspace{-0.4cm}

\cventry
{Co-Founder \& Senior Consultant}
{WiSolve Consulting}
{Madison, WI}
{03/2016--present}
{}
\vspace{-0.4cm}
\cventry
{Director of Technology}
{}
{}
{01/2018--09/2019}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Provided business services (including market research analysis, business plan development, competitive landscape analysis, SBIR grant writing, and others) to early-stage companies in the biotech and pharmaceutical industries.}
%\end{cvitems}
}

\vspace{-0.4cm}
  
\cventry
{Postdoctoral Associate - Department of Bacteriology - Advisor: Cameron R Currie, PhD}
{University of Wisconsin-Madison}
{Madison, WI}
{04/2019--05/2019}
{}
\vspace{-0.4cm}
\cventry
{PhD Candidate - Department of Genetics - Advisor: Cameron R Currie, PhD}
{}
{}
{08/2015--04/2019}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Built genomics-driven computational and analytic pipelines to uncover novel therapeutics and study the evolution of biosynthesis in free-living and host-associated microbes.}
%\item {Systems investigated include insects (leaf-cutting ants, honey bees, \& insects broadly), marine invertebrates, soil communities, \& the human microbiome.}
%\item {Led computational arm of Wisconsin Antimicrobial Drug Discovery \& Development Center (NIH CETR).}
%\end{cvitems}
}

\vspace{-0.4cm}
      
\cventry
{Lead Computational Biologist - Departments of Biology and Planetary Science - Advisor: Sarah S Johnson, PhD}
{Harvard \& Georgetown Universities}
{Cambridge, MA}
{10/2013--10/2015}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Performed whole genome sequencing and metagenomic analysis of environmental samples from sulfur-rich, extreme environments with implications in microbial ecology, biogeochemistry, and exobiology.}
%\item {Characterized biosynthetic potential of metagenomic data.}
%\end{cvitems} 
}

\vspace{-0.4cm}

\cventry
{Head of Experimental Genomics}
{Warp Drive Bio}
{Cambridge, MA}
{04/2013--08/2015}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Executed genomic-directed natural products drug discovery, high throughput Next Generation Sequencing (htNGS), computational biology, and molecular biology of actinomycetes and fungi.}
%\item {Designed and implemented genomic natural products searches over various scaffolds of business development and internal interest.}
%\item {Developed and curated computational pipelines and databases for assembly, annotation, and custom analysis of public and internal htNGS data (160,000 bacterial genomes, >150 closed and complete genomes) for analysis of novel polyketide, non-ribosomal peptide, and other natural product classes.}
%\item {Handled processing and management of sequence data, predictions, and analyses supporting multiple projects across discovery, molecular biology, engineering, and synthetic biology.}
%\item {Executed elucidation and prediction of novel chemical products of bacterial biosynthetic gene clusters and metabolic pathways (e.g. beta-lactams, aminoglycosides, rapamycin analogues, etc.).}
%\item {Developed internal pipelines for applied phylogenomic annotations and prioritizations of multiple data types to inform discovery and engineering efforts.}
%\item {Oversaw all lab and experimental support of actinomycete and fungal sequencing efforts for Illumina, Pacific Biosciences, and Oxford-Nanopore platforms.}
%\item {Bioinformatics software development to support molecular and synthetic biology efforts.}
%\item {Direct written and verbal communication of findings to senior leadership and business partners.}
%\item {Database management and delivery of sequence information to molecular biology, microbiology, and chemistry groups to aid drug discovery, strain engineering, and generation of expression constructs.}
%\end{cvitems}
}

\vspace{-0.4cm}

\cventry
{Research Assistant, Microbiology \& Computational Biology - Advisor: Tom\'{a}s Maira-Litr\'{a}n, PharmD, PhD}
{Brigham \& Women's Hospital}
{Boston, MA}
{03/2013--08/2015}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Investigated \textit{in vivo} fitness, horizontal gene transmission, and pathogenesis of \textit{Acinetobacter baumannii}, \textit{Staphylococcus aureus}, \textit{Salmonella typhii}, and other virulent pathogens through microbiology, computational, and genomic techniques.}
%\item {Developed and optimized genetic tools to enable novel examinations of pathogen fitness, invasion, and virulence using high-throughput transposon-directed insertion site sequencing of infections in murine models.}
%\end{cvitems}
}

\vspace{-0.4cm}
       
\cventry
{Research Associate II, Molecular Biology Process Development}
{Broad Institute of MIT \& Harvard}
{Cambridge, MA}
{01/2011--03/2013}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Independently designed development initiatives including supporting htNGS, microfluidics, and automation goals.}
%\item {Oversaw production and up-scaling of microbial mate-pair library construction (LC), integrated internal development with vendor technologies, and managed sample-tracking via real-time messaging to internal LIMS.}
%\item {Increased throughput of microbial LC Platform 4-fold by automation and protocol development.}
%\item{Worked extensively with mate-pair NGS LC, sequence analysis tools, genomic databases, statistical software, and programming/operating lab robotics.}
%\end{cvitems}
}

\vspace{-0.4cm}

\cventry
{Research Associate, Molecular Genetics - Advisor: Eric Rutledge, PhD}
{Rensselaer Polytechnic Institute}
{Troy, NY}
{05/2010--12/2010}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Designed and developed protocols and operating procedures for transgenic \textit{Caenorhabditis elegans} cultures to model stress-induced neural degeneration and Parkinson's Disease.}
%\end{cvitems}
}

\vspace{-0.4cm}
       
\cventry
{Research Assistant, Microbiology}
{BCR Biotech}
{Jamestown, RI}
{09/2009--12/2009}
{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Wrote and optimized protocols and methods for engineering synthetic biosensing functions in \textit{Bacillus} spores.}
%\end{cvitems}
}

\end{cventries}

\vspace{-7mm}